egs/thchs30/s5/local/dae/add-noise-mod.py

```diff
@@ -0,0 +1,157 @@
+#!/nfs/disk/perm/linux-x86_64/bin/python2
+###!/usr/bin/python
```

Comment:
I suggest using only shebang
#!/usr/bin/env python

Everything else complicates running the scripts elsewhere. The situation is complicated enough with Py2.x and Py3